client/src/components/ImageStrip/ImageStrip.jsx

```diff
@@ -3,11 +3,16 @@ import GamesContext from '../../context';
 import { CarouselImage } from './styles';
 
 const ImageStrip = ({ image }) => {
-  const { setMainImage } = useContext(GamesContext);
+  const { setMainImage, setImageFade, imageFade } = useContext(GamesContext);
+  // let hi = 1;
```

Comment:
This is probably old code you've commented out that can just be deleted? Unless you're saving it for something in the future?